psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -1264,9 +1264,9 @@ void HF::print_orbitals()
                 orb_order[orb_e[a].second] = a;
 
             for (int a = 0; a < nalphapi_[h]; a++)
-                occ.push_back(std::make_pair(epsilon_a_->get(h,a), std::make_pair(labels[h],orb_order[a] + 1)));
+                occ.push_back(std::make_pair(epsilon_a_->get(h,a), std::make_pair(labels[h].c_str(),orb_order[a] + 1)));
```

Comment:
I don't see any reason to keep using `std::pair<const char*, int>` can probably just  switch to std::pair<const std::string, int>.

There would be no need to use `labels[h].c_str()` all over the place in `HF::print_orbitals()`. you would only need to using it once in the overload above (after changing the argument types too)